blocks/math.js

```diff
@@ -32,32 +32,24 @@ goog.require('Blockly.Colours');
 
 goog.require('Blockly.constants');
 
-
-/**
- * Common HSV hue for all blocks in this category.
- */
-Blockly.Blocks.math.HUE = Blockly.Colours.textField;
-
 Blockly.Blocks['math_number'] = {
   /**
    * Block for generic numeric value.
    * @this Blockly.Block
    */
   init: function() {
```

Comment:
By "our shared blocks" do you mean "blocks shared between vertical and horizontal" or "blocks shared between google/blockly and scratch-blocks"?  If it's the former--yes, I think that would be a good idea.